src/ape/managers/chain.py

```diff
@@ -1111,7 +1111,17 @@ def instance_at(
                 raise  # Current exception
 
         if not contract_type:
-            raise ChainError(f"Failed to get contract type for address '{contract_address}'.")
+            msg = f"Failed to get contract type for address '{contract_address}'."
+            if self.provider.network.explorer is None:
+                msg += (
+                    f" Current provider '{self.provider.name}' has no associated "
+                    "explorer plugin. Try installing an explorer plugin using "
+                    f"{Colors.GREEN}ape plugins install etherscan{Colors.END}, "
```

Comment:
You should `click.style` instead maybe, then we wouldnt need to define all the ansi colors

Reply:
or alternatively, the rich library all over will do stuff like this automatically , or you can control with CLI-markup